addons/keybinding/fnc_addKeybind.sqf

```diff
@@ -88,6 +88,10 @@ if (_defaultKeybind isEqualTypeParams [0, false, false, false]) then {
     _defaultKeybind = [_defaultKey, [_defaultShift, _defaultControl, _defaultAlt]];
 };
 
+// Handle empty strings to indicated no code
+if (_downCode isEqualType "") then {_downCode = {};};
+if (_upCode isEqualType "") then {_upCode = {};};
```

Comment:
I know what you mean, but then this isn't limited to STRING either, but any type.
And `"#nothing"` is a bad example, because that actually doesn't throw an error when compiling or calling the result, because `#` is treated like `#LINE 1` and other instructions that survive preprocessing even though it does nothing.